src/main/java/ch/njol/skript/localization/Language.java

```diff
@@ -135,132 +104,149 @@ public static String get(final String key) {
-		if (din == null)
-			throw new IllegalStateException(addon + " is missing the required english.lang file!");
-		HashMap<String, String> en;
-		try {
-			en = new Config(din, "english.lang", false, false, ":").toMap(".");
-		} catch (final Exception e) {
-			throw Skript.exception(e, "Could not load " + addon + "'s default language file!");
-		} finally {
-			try {
-				din.close();
-			} catch (final IOException e) {}
+
+		InputStream din = addon.plugin.getResource(addon.getLanguageFileDirectory() + "/default.lang");
+		if (din == null && addon != Skript.getAddonInstance()) {
+			// Backwards compatibility with addons
+			din = addon.plugin.getResource(addon.getLanguageFileDirectory() + "/english.lang");
 		}
-		final String v = en.get("version");
+
+		if (din == null)
+			throw new IllegalStateException(addon + " is missing the required default.lang file!");
+
+		HashMap<String, String> en = load(din, "default");
+
+		String v = en.get("version");
 		if (v == null)
-			Skript.warning("Missing version in english.lang");
+			Skript.warning("Missing version in default.lang");
+
 		langVersion.put(addon.plugin, v == null ? Skript.getVersion() : new Version(v));
 		en.remove("version");
-		english.putAll(en);
-		for (final LanguageChangeListener l : listeners)
+		defaultLanguage.putAll(en);
+
+		for (LanguageChangeListener l : listeners)
 			l.onLanguageChange();
 	}
 	
 	public static boolean load(String name) {
 		name = "" + name.toLowerCase();
-		if (name.equals("english"))
-			return true;
-		localized = new HashMap<>();
+
+		localizedLanguage = new HashMap<>();
 		boolean exists = load(Skript.getAddonInstance(), name);
-		for (final SkriptAddon addon : Skript.getAddons()) {
-			assert addon != null;
+		for (SkriptAddon addon : Skript.getAddons()) {
 			exists |= load(addon, name);
 		}
 		if (!exists) {
-			localized = null;
-			Language.name = "english";
+			if (name.equals("english")) {
+				throw new RuntimeException("English language is missing (english.lang)");
+			} else {
+				load("english");
+			}
 			return false;
 		}
+
 		Language.name = name;
-		validateLocalized();
-		if (useLocal) {
-			for (final LanguageChangeListener l : listeners)
-				l.onLanguageChange();
-		}
+
+		for (LanguageChangeListener l : listeners)
+			l.onLanguageChange();
+
 		return true;
 	}
 	
-	private static boolean load(final SkriptAddon addon, final String name) {
+	private static boolean load(SkriptAddon addon, String name) {
 		if (addon.getLanguageFileDirectory() == null)
 			return false;
-		final HashMap<String, String> l = load(addon.plugin.getResource(addon.getLanguageFileDirectory() + "/" + name + ".lang"), name);
-		final File f = new File(addon.plugin.getDataFolder(), addon.getLanguageFileDirectory() + File.separator + name + ".lang");
+		// Backwards addon compatibility
+		if (name.equals("english") && addon.plugin.getResource(addon.getLanguageFileDirectory() + "/default.lang") == null)
```

Comment:
Should everything be using `File.separator` in place of just `/`?

Reply:
Turns out zip files should always have forward slashes:
![image](https://user-images.githubusercontent.com/29547183/123604259-f8bf1400-d7fa-11eb-80f0-860dde1578b2.png)